### PR TITLE
[DOC-3258] Update third party tool and sdk versions for delegate

### PR DESCRIPTION
**Pull request by brian-f-harness**
Update third party tool and sdk versions for delegate

For reviewers, previews are available:

[Third party tools installed with the delegate image type](https://64951a4d902ad12e3ac2ac71--harness-developer.netlify.app/docs/platform/Delegates/delegate-concepts/delegate-image-types#third-party-tools-included-in-the-delegate-image-type)
[SDKs installed with Harness Delegate](https://64951a4d902ad12e3ac2ac71--harness-developer.netlify.app/docs/getting-started/supported-platforms-and-technologies#sdks-installed-with-harness-delegate)

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/2_Delegates/delegate-concepts/delegate-image-types.md [hunk 1]
```diff
@@ -31,8 +31,8 @@ The following table differentiates between delegate images based on key features
 | DELEGATE-LEGACY<br /><br />**Deprecated**: Not recommended for use in new Harness accounts | &#x2713; | x | &#x2713; | x | |
 
 ## Third-party tools included in the DELEGATE image type
-  
-  | **Third-party tool** | **78101 and earlier** | **78306 and later** |
+
+  | **Third-party tool/SDK** | **78101 and earlier** | **78306 and later** |
```
Comment (bharatgoelharness) [inline docs/platform/2_Delegates/delegate-concepts/delegate-image-types.md:35]: I think we can remove this condition now. 
@pratmit @raghuAtWings As we are on 799 and next month we will be releasing 800xx series of delegate.

Comment (pratmit) [reply docs/platform/2_Delegates/delegate-concepts/delegate-image-types.md:35]: We can let this information for some more time. This helps to drive the conversation with the customers and show that we are updating the 3P binaries on a continuous basis.

**Review by pratmit — DISMISSED** [review]
Conditionally approve based on the assumption that the versions are correct for the 3P binaries.

**Comment by brian-f-harness** [comment]
> Conditionally approve based on the assumption that the versions are correct for the 3P binaries.

Thank you, @pratmit. @bharatgoelharness Thank you for your review. Are the versions correct for 3P binaries? Upon your approval, I send the PR for peer review.

**Review by thisrohangupta — APPROVED** [review]

**Review by michaelcretzman — APPROVED** [review]

**Pull request merged**